.travis.scripts/compile.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export CFLAGS="-Wall -Wextra -Wdeclaration-after-statement -Wmissing-field-initializers -Wshadow -Wno-unused-parameter -ggdb3"
+export CFLAGS="-Wall -Wextra -Wdeclaration-after-statement -Wmissing-field-initializers -Wshadow -Wno-unused-parameter -ggdb3 -DDEBUGGER_FAIL_SILENTLY"
```

Comment:
Yes, it's not the nicest thing on earth but a Little nicer than defining it in the c file ;-)